src/content.ts

```diff
@@ -1181,6 +1181,7 @@ function startSponsorClicked() {
     if (sponsorTimesSubmitting.length > 0 && sponsorTimesSubmitting[sponsorTimesSubmitting.length - 1].segment.length < 2) {
         //it is an end time
         sponsorTimesSubmitting[sponsorTimesSubmitting.length - 1].segment[1] = getRealCurrentTime();
+        sponsorTimesSubmitting[sponsorTimesSubmitting.length - 1].segment.sort((a, b) => a > b ? 1 : (a < b ? -1 : 0));
```

Comment:
Is there a reason not to just do the following?

```
sponsorTimesSubmitting[sponsorTimesSubmitting.length - 1].segment.sort();
```

Reply:
.sort() is converting all values to string and does a UTF-16 codepoint comparing

`[0.5533,2.4887,20.5338,50.4557,10.77778].sort()`
`outputs: [0.5533, 10.77778, 2.4887, 20.5338, 50.4557]`

Possible is `.sort((a,b)=>a-b)`  
`output: [0.5533, 2.4887, 10.77778, 20.5338, 50.4557]`

Reply:
Wow, JavaScript really is wonderful